docs/examples/quick-start-with-terraform.md

```diff
@@ -0,0 +1,302 @@
+The Liqo webhook will add the toleration for this taint to the pods created in the liqo-enabled namespaces.
+```
+
+Since no further configuration is provided, Liqo will add a suffix to the namespace name to make it unique on the remote cluster (see the dedicated [usage page](/usage/namespace-offloading.md) for additional information concerning namespace offloading configurations).
+
+### Deploy simple pod on offloaded namespace
+
+At this point the basic setup is completed and next resources created by Terraform will deploy a standard Kubernetes application in a multi-cluster environment as you would do in a single cluster scenario.
+
+The deployed resources are t two pods running an `nginx` image and a Service exposing the pods to the cluster.
+One pods is running in the local cluster, while the other is forced to be scheduled on the remote cluster.
+
+```{admonition} Info
+Differently from the traditional examples, the above deployment introduces an *affinity* constraint. This forces Kubernetes to schedule the first pod (i.e. `nginx-local`) on a physical node and the second (i.e. `nginx-remote`) on a virtual node.
+Virtual nodes are like traditional Kubernetes nodes, but they represent remote clusters and have the `liqo.io/type: virtual-node` label.
+
+When the affinity constraint is not specified, the Kubernetes scheduler selects the best hosting node based on the available resources.
+Hence, each pod can be scheduled either in the *local* cluster or in the *remote* cluster.
+```
+
+Now you can check the status of the pods.
+The output should be similar to the one below, confirming that one `nginx` pod is running locally; while the other is hosted by the virtual node (i.e., `liqo-milan`).
+
+```bash
+kubectl get pod -n liqo-demo -o wide
+```
+
+And the output should look like this:
+
+```text
+NAME           READY   STATUS    RESTARTS   AGE   IP            NODE          NOMINATED NODE   READINESS GATES
+nginx-local    1/1     Running   0          94s   10.200.1.10   rome-worker   <none>           <none>
+nginx-remote   1/1     Running   0          94s   10.202.1.9    liqo-milan    <none>           <none>
+```
+
+#### Check the pod connectivity
+
+Once both pods are correctly running, it is possible to check one of the abstractions introduced by Liqo.
+Indeed, Liqo enables each pod to be transparently contacted by every other pod and physical node (according to the Kubernetes model), regardless of whether it is hosted by the _local_ or by the _remote_ cluster.
+
+First, let's retrieve the IP address of the `nginx` pods:
+
+```bash
+LOCAL_POD_IP=$(kubectl get pod nginx-local -n liqo-demo --template={{.status.podIP}})
+REMOTE_POD_IP=$(kubectl get pod nginx-remote -n liqo-demo --template={{.status.podIP}})
+echo "Local Pod IP: ${LOCAL_POD_IP} - Remote Pod IP: ${REMOTE_POD_IP}"
+```
+
+You can fire up a pod and run `curl` from inside the cluster:
```

Comment:
```suggestion
You can fire up a _client_ pod and run `curl` from inside the cluster:
```

Reply:
This section is removed following Marco Iorio's suggestions.